docs/input/docs/reference/version-increments.md

```diff
@@ -95,9 +95,9 @@ you can leverage this feature as follows:
 
 ```yaml
 mode: MainLine # Only add this if you want every version to be created automatically on your main branch.
-major-version-bump-message: "^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(\\([\\w\\s]*\\))?(!:|:.*\\n\\n((.+\\n)+\\n)?BREAKING CHANGE:\\s.+)"
-minor-version-bump-message: "^(feat)(\\([\\w\\s]*\\))?:"
-patch-version-bump-message: "^(build|chore|ci|docs|fix|perf|refactor|revert|style|test)(\\([\\w\\s]*\\))?:"
+major-version-bump-message: "^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(\\([\\w\\s-]*\\))?(!:|:.*\\n\\n((.+\\n)+\\n)?BREAKING CHANGE:\\s.+)"
+minor-version-bump-message: "^(feat)(\\([\\w\\s-]*\\))?:"
```

Comment:
Can you please update the regex in the accompanying test as well?

https://github.com/GitTools/GitVersion/blob/9199d7b0b2afbe961f7776592e9afb24707b2842/src/GitVersion.Core.Tests/IntegrationTests/VersionBumpingScenarios.cs#L82

Reply:
Ok, I will do that. Will clone the repo this time :)

Reply:
you can try codespaces